COPYING

```diff
@@ -39,245 +39,79 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-----------------------------------------------------------------------
 
-CVC4 incorporates MiniSat code, excluded from the above copyright.
-See src/sat/minisat.  Its copyright:
+-------------------------------------------------------------------------------
+ Third-Party Software
+-------------------------------------------------------------------------------
 
-  MiniSat -- Copyright (c) 2003-2006, Niklas Een, Niklas Sorensson
-             Copyright (c) 2007-2010  Niklas Sorensson
+CVC4 incorporates third-party software without changing the terms above.
```

Comment:
The CVC4 source code includes third-party software which has its own copyright and licensing terms, as described below.